generate-src-docs.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+rm -rf src-docs
+lazydocs --no-watermark --output-path src-docs src/*.py
```

Comment:
Out of curiosity, what is the reason for having *.py please? wouldn't src/* be enough?

Reply:
it is so that each file is considered separately rather than together - mostly this is for the charm rather than in this case